tools/flang2/flang2exe/lldebug.cpp

```diff
@@ -2008,23 +2064,27 @@ lldbg_emit_outlined_subprogram(LL_DebugInfo *db, int sptr, int findex,
   mips_linkage_name = func_name;
   is_def = DEFDG(sptr);
   is_def |= (STYPEG(sptr) == ST_ENTRY);
+  if (ll_feature_has_diextensions(&db->module->ir))
+    flags = set_disubprogram_flags(sptr);
```

Comment:
I think it would be clearer if `set_disubprogram_flags` was customized to omit the DIFLAGs that are no longer in that flags word rather than introducing more pseudo-constants and relying on them being reset to 0.